website/content/docs/plugins/creation/index.mdx

```diff
@@ -212,89 +167,64 @@ Here's what you need to create releases using GitHub Actions:
    to install a plugin using `packer init`, see the
-   [init docs](/packer/docs/commands/init).
-
-## Registering Plugin Documentation
-
-~> Note: Registering a remote plugin's plugin documentation requires the use of [Packer's plugin docs configuration](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
-
-`packer init` allows users to require and install remote Packer plugins, those not bundled with Packer core, that have been published to GitHub automatically.
-To help with the discovery of remote Packer plugins on GitHub, plugins maintainers can choose to register plugin documentation for each component directly on the [Packer Documentation Page](/packer/docs).
-
-The registration process requires the creation of a `docs.zip` file archive containing the `.mdx` files for each of the plugin components in the remote plugin's repository. A working example can be seen at the [packer-plugin-docker repository](https://github.com/hashicorp/packer-plugin-docker/releases/latest).
-
-Once in place the remote plugin can be added to Packer's website builds by opening a pull-request against [hashicorp/packer](https://github.com/hashicorp/packer), with the needed configuration for pulling in the remote documentation.
-
-Remote plugins will have their components listed under the respected types (i.e builders, provisioners, etc) using the names specified in the remote block configuration, and labeled with their respective [tier and namespace](/packer/docs/plugins#tiers-and-namespaces).
-
-To register a plugin follow one of the following setups
-
-<Tabs>
-<Tab heading="Generating docs.zip on plugin release">
-
-Documentation for a plugin is maintained within the `docs` directory and served on GitHub.
-
-To include plugin docs on the website, a global pre-hook has been added to the main scaffolding [.goreleaser.yml](https://github.com/hashicorp/packer-plugin-scaffolding/blob/42e5b0b1e575879b0477cb6d4291e027f4d92f85/.goreleaser.yml#L10) file, that if uncommented will generate and include a docs.zip file as part of the plugin release.
-
-The `docs.zip` file contains all of the `.mdx` files under the plugins root `docs/` directory that the website can consume remotely.
-
-</Tab>
-
-<Tab heading="Manually generating docs.zip">
-
-You can generate the required documentation structure manually by creating a zip file called `docs.zip` of the docs directory and including that zip file in the plugin release.
-
-```/bin/bash
-[[ -d docs/ ]] && zip -r docs.zip docs/
+   [init docs](https://developer.hashicorp.com/packer/docs/commands/init).
+
+## Registering Plugins
+
+~> Note: Registering a plugin as an integration requires the use of [Packer's Incofiguration configuration](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/docs).
```

Comment:
Not sure what the exact intent was here, but I believe we only require the final hierarchy to match what's in `.web-docs`, how a developer structures their code doesn't matter, so long as we can get this from the git repo.

```suggestion
~> Note: Registering a plugin as an integration requires the documentation to match the [Scaffolding example layout](https://github.com/hashicorp/packer-plugin-scaffolding/tree/main/.web-docs).
```

The link will be dead until we merge https://github.com/hashicorp/packer-plugin-scaffolding/pull/108 too

Reply:
Also, since we state below that the directory structure must respect a convention (where do we document it?), we could also do away with this note, and add the link to the scaffolding's `.web-docs` in here